tests/test_pqserver.py

```diff
@@ -142,24 +124,20 @@ def app(db, scope="module"):
         pqserver.engine.dispose()
 
 
-@pytest.fixture
-def client(app, scope="module"):
```

Comment:
Clarification: We remove this because every endpoint is so dependent on everything being set up like in the lsstcomcamsim fixture, so we cannot test anything with bare client() ?

Reply:
Correct.